src/main/kotlin/no/nav/familie/ba/sak/annenvurdering/AnnenVurderingService.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ba.sak.annenvurdering
+
+import no.nav.familie.ba.sak.behandling.restDomene.RestAnnenVurdering
+import no.nav.familie.ba.sak.behandling.vilkår.PersonResultat
+import no.nav.familie.ba.sak.logg.LoggService
+import no.nav.familie.ba.sak.nare.Resultat
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class AnnenVurderingService(
+        private val annenVurderingRepository: AnnenVurderingRepository,
+) {
+
+    fun hent(personResultat: PersonResultat, andreVurderingerType: AnnenVurderingType): AnnenVurdering? =
+            annenVurderingRepository.findBy(personResultat = personResultat,
+                                            type = andreVurderingerType)
+
+    fun hent(annenVurderingId: Long): AnnenVurdering = annenVurderingRepository.findById(annenVurderingId)
+            .orElseThrow { error("Annen vurdering med id $annenVurderingId finnes ikke i db") }
+
+    @Transactional
+    fun endreAnnenVurdering(annenVurderingId: Long, restAnnenVurdering: RestAnnenVurdering) {
+        hent(annenVurderingId = annenVurderingId).let {
+            annenVurderingRepository.save(it.also {
+                it.resultat = restAnnenVurdering.resultat
+                it.begrunnelse = restAnnenVurdering.begrunnelse
+                it.type = restAnnenVurdering.type
+            })
+        }
+    }
+}
+
+fun PersonResultat.leggTilBlankAnnenVurdering(andreVurderingerType: AnnenVurderingType) {
```

Comment:
```suggestion
fun PersonResultat.leggTilBlankAnnenVurdering(annenVurderingType: AnnenVurderingType) {
```

siden det er en public extension av personresultat, bør den kanskje ligge på personresultat forresten? der ligger også extensionfunksjonene for å mutere vilkårresultatene på det